src/server.py

```diff
@@ -395,14 +396,18 @@ def run_create_jobtypes_query():
             for testtype in cursor.fetchall():
                 testtype = testtype[0]
                 # ignore talos, builds, jetpack
-                if testtype in ['svgr', 'svgr-e10s', 'svgr-snow', 'svgr-snow-e10s',
-                                'other', 'other-e10s', 'other-snow', 'other-snow-e10s',
-                                'chromez', 'chromez-e10s', 'chromez-snow', 'chromez-snow-e10s',
+                if testtype in ['svgr', 'svgr-e10s', 'svgr-osx', 'svgr-osx-e10s',
+                                'other', 'other-e10s', 'other-osx', 'other-osx-e10s',
+                                'chromez', 'chromez-e10s', 'chromez-osx', 'chromez-osx-e10s',
                                 'tp5o', 'tp5o-e10s', 'dromaeojs', 'dromaeojs-e10s',
-                                'g1', 'g2', 'g2-e10s', 'g1-e10s', 'g1-snow', 'g1-snow-e10s',
+                                'g1', 'g2', 'g2-e10s', 'g1-e10s', 'g1-snow', 'g1-osx-e10s',
                                 'other_nol64', 'other_nol64-e10s', 'other_l64', 'other_l64-e10s',
-                                'xperf', 'xperf-e10s', 'dep', 'nightly', 'jetpack',
-                                'non-unified', 'valgrind', '5151c298eaed59034a45e3c4e3d4e0003fed4a14']:
+                                'xperf', 'xperf-e10s', 'other-e10s', 'dep', 'nightly', 'jetpack',
+                                'non-unified', 'valgrind']:
+                    continue
+
+                # skip taskcluster jobs
+                if len(testtype) == 40:
```

Comment:
I think this condition is not really required. What if the testtype for a non-taskcluster job is 40? It will then skip it even though we don't want that. Since there is no "b2g" in the platforms, I think this condition is unnecessary, let me know if I am missing something here.